tests/CMakeLists.txt

```diff
@@ -314,6 +314,9 @@ if(WITH_PYTHON)
   add_fdb_test(
     TEST_FILES restarting/from_7.0.0_until_7.1.0/UpgradeAndBackupRestore-1.toml
     restarting/from_7.0.0_until_7.1.0/UpgradeAndBackupRestore-2.toml)
+  add_fdb_test(
+    TEST_FILES restarting/from_7.1.0_until_7.2.0/ClientMetricRestart-1.toml
```

Comment:
7.1.0 should be replaced with the actual version number that has the workload.

Reply:
Yes that is true, I also noted that in PR description, thanks for pointing out